recipes/fast-cdr/all/patches/2.0.0-0001-Fix-for-non-CWG-1270-revision-compliant-compilers-17.patch

```diff
@@ -0,0 +1,33 @@
+From fe9a125ba5809fc1f7aa5799d4285392e837f35e Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Utkan=20K=C4=B1l=C4=B1=C3=A7?=
+ <65946928+utkankilic@users.noreply.github.com>
+Date: Wed, 25 Oct 2023 08:33:08 +0300
+Subject: [PATCH] Fix for non CWG 1270 revision compliant compilers (#173)
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Signed-off-by: Utkan Kılıç utkankilic@gmail.com
+
+Signed-off-by: utkan.kilic <utkan.kilic@togg.com.tr>
+Co-authored-by: utkan.kilic <utkan.kilic@togg.com.tr>
+---
+ include/fastcdr/Cdr.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/include/fastcdr/Cdr.h b/include/fastcdr/Cdr.h
+index 254cd08..9d08820 100644
+--- a/include/fastcdr/Cdr.h
++++ b/include/fastcdr/Cdr.h
```

Comment:
@ma30002000 Could you please send this patch to the upstream? C++ portability should not be managed by Conan, because is not tested/validated by the upstream. 

https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/sources_and_patches.md#policy-on-patches

Reply:
Thank you!